eip-0034.md

```diff
@@ -0,0 +1,121 @@
+# NFT Collection Standard V1
+
+* Author: anon_real
+* Status: Proposed
+* Created: 10-May-2022
+* License: CC0
+* Forking: not needed 
+
+## Motivation 
+NFT collections play a central role in the nowadays NFT ecosystem on all blockchains. It gives the ability to artists to organize their artworks and present them in a more meaningful way. It also can facilitate the use of other NFT features such as traits. As we'll present in this proposal, it can also enable more advanced features such as artist collaborations. These advanced features can get arbitrary complex with the use of smart contracts.
```

Comment:
```suggestion
NFT collections play a central role in the nowadays NFT ecosystem on all blockchains. It gives the ability to artists to organize their artworks and present them in a more meaningful way. It also can facilitate the use of other NFT features such as traits. As we'll present in this proposal, it can also enable more advanced features such as artist collaborations. These advanced features can get arbitrary complex with the use of smart contracts.
```
```suggestion
NFT collections play a central role in the nowadays NFT ecosystem on all blockchains. It gives the ability to artists to organize their artworks and present them in a more meaningful way. It also can facilitate the use of other NFT features such as traits. As we'll present in this proposal, it can also enable more advanced features such as artist collaborations. These advanced features can get arbitrarily complex with the use of smart contracts.
```